runtime/src/accounts_db.rs

```diff
@@ -451,11 +451,40 @@ pub struct AccountsDB {
     /// Thread pool used for par_iter
     pub thread_pool: ThreadPool,
 
+    pub thread_pool_clean: ThreadPool,
+
     /// Number of append vecs to create to maximize parallelism when scanning
     /// the accounts
     min_num_stores: usize,
 
     pub bank_hashes: RwLock<HashMap<Slot, BankHashInfo>>,
+
+    pub dead_slots: RwLock<HashSet<Slot>>,
+}
+
+fn make_min_priority_thread_pool() -> ThreadPool {
+    use thread_priority::{
+        set_thread_priority, thread_native_id, NormalThreadSchedulePolicy, ThreadPriority,
+        ThreadSchedulePolicy,
+    };
+    let num_threads = get_thread_count();
+    rayon::ThreadPoolBuilder::new()
+        .spawn_handler(|thread| {
+            std::thread::spawn(|| {
+                let thread_id = thread_native_id();
+                set_thread_priority(
+                    thread_id,
+                    ThreadPriority::Min,
+                    ThreadSchedulePolicy::Normal(NormalThreadSchedulePolicy::Normal),
+                )
+                .unwrap();
```

Comment:
[nits] I think this code block can be extracted to a newly-defined handler by hooking to [`.start_handler`](https://docs.rs/rayon/1.3.0/rayon/struct.ThreadPoolBuilder.html#method.start_handler). So, we can avoid writing our own spawning logic, which slightly different to [the default](https://docs.rs/rayon/1.3.0/rayon/struct.ThreadPoolBuilder.html#method.start_handler): 

> The default spawn handler sets the name and stack size if given, and propagates any errors from the thread builder.
> 
> ```
>     let pool = rayon::ThreadPoolBuilder::new()
>         .spawn_handler(|thread| {
>             let mut b = std::thread::Builder::new();
>             if let Some(name) = thread.name() {
>                 b = b.name(name.to_owned());
>             }
>             if let Some(stack_size) = thread.stack_size() {
>                 b = b.stack_size(stack_size);
>             }
>             b.spawn(|| thread.run())?;
>             Ok(())
>         })
>         .build()?;
> ```

Especially, I'd like newly-spawned threads to be named for debugging purpose.